clients/fides-js/src/lib/consent-utils.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Wrapper around 'console.log' that only logs output when the 'debug' banner
+ * option is truthy
+ */
+type ConsoleLogParameters = Parameters<typeof console.log>;
+export const debugLog = (
+  enabled: boolean,
+  ...args: ConsoleLogParameters
+): void => {
+  if (enabled) {
+    // eslint-disable-next-line no-console
+    console.log(...args); // TODO: use console.debug instead?
```

Comment:
TODO here- what do y'all think?

Reply:
I vote stick with `console.log`. We already don't log anything unless `options.debug: true` is passed in to `Fides.init()`. 

In addition, `console.debug` is not enabled by default in browsers, which raises barriers (OK, slightly 😉 ) for ease of dev/testing